lib/logstash/outputs/elasticsearch/http_client/manticore_adapter.rb

```diff
@@ -1,4 +1,5 @@
 require 'manticore'
+require_relative '../../../../manticore/patches/client'
```

Comment:
Generally a `require_relative` with more than one `../` in it is a warning sign to me that the file we are requiring probably isn't as well-scoped as it should be.